Microsoft.Toolkit.Uwp/Helpers/BluetoothLEHelper/BluetoothLEHelper.cs

```diff
@@ -231,16 +247,11 @@ public void StopEnumeration()
         {
             ObservableBluetoothLEDevice device = null;
 
-            if (_readerWriterLockSlim.TryEnterUpgradeableReadLock(TimeSpan.FromSeconds(1)))
-            {
-                device = BluetoothLeDevices.FirstOrDefault(i => i.DeviceInfo.Id == deviceInfoUpdate.Id);
-
-                if (device != null)
-                {
-                    await device.Update(deviceInfoUpdate);
-                }
+            device = BluetoothLeDevices.FirstOrDefault(i => i.DeviceInfo.Id == deviceInfoUpdate.Id);
```

Comment:
by design. If the device is removed while iterating, then it's ok to be null. If we hold on to it then it will cause an exception when I try to update it, and I actually don't care about updating a removed object.